src/sentry/web/frontend/organization_auth_settings.py

```diff
@@ -73,6 +73,10 @@ def _disable_provider(self, request, organization, auth_provider):
                 ),
             )
 
+        user_ids = OrganizationMember.objects.filter(organization=organization).only('user__id')
+        User.objects.filter(id__in=user_ids).update(is_managed=False)
```

Comment:
This compiles to

```sql
UPDATE "auth_user" SET "is_managed" = false WHERE "auth_user"."id" IN (SELECT U0."id" FROM "sentry_organizationmember" U0 WHERE U0."organization_id" = 3 )
```

Reply:
Updated this to use `values('user')` by @dcramer's recomendation. Compiled query is still the same.